api/tests/test_admin.py

```diff
@@ -0,0 +1,46 @@
+from django.contrib.admin.sites import AdminSite
+from django.test import TestCase
+from django.test import RequestFactory
+from api.models import Shipment
+from api.admin import ShipmentAdmin
+
+
+class MockSuperUser:
+    def has_perm(self, perm):
+        return True
+
+
+request_factory = RequestFactory()
+request = request_factory.get('/admin')
+request.user = MockSuperUser()
+
+
+class ShipmentAdminTest(TestCase):
+
+    def setUp(self):
+        Shipment.objects.all().delete()
+        self.shipment = Shipment.objects.create(
+            delivery_id="111",
+            order_id=2,
+            vendor_id=5,
+            label_url="my_pdf_url",
+            status="",
+        )
+        site = AdminSite()
+        self.admin = ShipmentAdmin(Shipment, site)
+
+    def test_shipment_actions(self):
+        actions = self.admin.shipment_actions(self.shipment)
+        self.assertEqual(
+            actions,
+            '<a class="button" href="/admin/api/shipment/' +
```

Comment:
Maybe it would be more pythonized to use: 
'<a class="button" href="/admin/api/shipment/{}'.format(self.shipment.pk) +
            '/delivery/retry/">Reintentar enviament</a>'
          
for concatenating strings or the python3 new way:
f'<a class="button" href="/admin/api/shipment/{self.shipment.pk}' +
            '/delivery/retry/">Reintentar enviament</a>'